### ISSUE TITLE
Verify the expectations of Name 1.0 combobox-focusable

### ISSUE THREAD

**Issue by joanmarie** [WhatSock/w3c-alternative-text-computation#11]
For this test case:
```
if given
  <div id="test" role="combobox" tabindex="0" title="Choose your language.">
    <span> English </span>
  </div>
then the accessible name of the element with id of "test" is "Choose your language. English"
```
I question the expected result which matches [what is in the WhatSock tool](https://whatsock.github.io/w3c-alternative-text-computation/Proposed%20Name%20and%20Description%20Tests/Name%201.0%20combobox-focusable.html). In particular, I don't think "English" should be included. Walking through the steps in the AccName algorithm, setting the current node to the div with id of "test":
* 2A doesn't apply because the current node isn't hidden.
* 2B doesn't apply because the current node lacks an aria-labelledby attribute
* 2C doesn't apply because the current node lacks an aria-label attribute
* 2D does apply, because the current node's native markup does provide an attribute (e.g. title) that defines a text alternative.

2D's instructions are: "return that alternative in the form of a flat string as defined by the host language, unless the element is marked as presentational (role="presentation" or role="none")." The alternative defined by the host language is the value of the title attribute, namely "Choose your language." So we return that attribute and end the calculation. This is, btw, what both Firefox and WebKit do.

**Comment by accdc** [comment]
This was actually being caused by a bug in title attribute processing that was fixed in 1.16, now still present in the latest version.
This one is complicated, because there is conflict between the 1.0 design pattern and the 1.1 where the name has to be treated differently for each.

E.G In the 1.0 design pattern where role=combobox is focusable, it has to support name from content when focusable, because if not, it will convey nothing at all if embedded text is included and there is no title or any other attribute on the focusable node like in the code above, which will make the visually displayed text impossible to read by screen readers. So, if it computes name from content, then name from content is always processed before the title attribute, such as a link with link text plus a title attribute to set the description. So in this case, the embedded text is the Name and the title attribute is the Description, which matches the same algorithm for link plus title in accordance with the naming computation.

For the 1.1 design pattern however, where focus is set inside of the combobox, then name from content must not be used to compute the name of the parent role=combobox, because doing so would clobber the name of the widget in the same manner as adding name from content to label role=region.

**Comment by joanmarie** [comment]
Having read the minutes from this week's concall and seeing the expectations having been changed in your tool, I've updated the wiki accordingly (fixed name expectations and added a description test case). Will update WPT soon.

Thanks for the change. Closing this one as fixed.
